src/App.js

```diff
@@ -1,10 +1,25 @@
+import MockMan from "mockman-js";
+import React from "react";
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
+import Footer from "./components/footer/Footer";
+import Navbar from "./components/navbar/Navbar";
```

Comment:
Try to use index.js in component folder and export all component from there, so that you can use import{Navbar,Footer} from "../component/index"

Reply:
thank you for this suggestion , it will look good and easy to import